_posts/2025-10-01-debezium-3-3-final-released.adoc

```diff
@@ -0,0 +1,829 @@
+
+
+[id=new-features-and-improvements-informix]
+=== Debezium for Informix
+
+==== Enhanced default value localization support
+
+The Informix connector now handles locale-specific database configurations more intelligently (https://issues.redhat.com/browse/DBZ-9181[DBZ-9181]).
+Instead of assuming a US locale, the connector properly parses locale-dependent values like `DBMONEY`, `DBCENTURY`, and `DBDATE` based on your actual database configuration.
+
+This improvement ensures more accurate data capture across diverse international deployments and eliminates potential data parsing errors in non-US environments.
+
+
+[id=new-features-and-improvements-server]
+=== Debezium Server
+
+==== Azure Event Hubs - Built-in hashing partition keys
+
+Azure Event Hub's partition keys are limited to a maximum of 128 characters.
+In the event that an event's key results in a partition key that exceeds that limit, Debezium Server's Azure Event Hubs sink adapter would throw a exception and the connector would terminate.
+This is less than desirable behavior and introduces constraints on source tables where it may not be possible to address at the source.
+
+To address this concern, the Azure Event Hubs sink can be configured to use one of several hashing algorithms to adhere to the maximum partition key size while working with source data where the partition key would otherwise exceed the maximum allowed size (https://issues.redhat.com/browse/DBZ-9245[DBZ-9245]).
+
+To enable the Azure Event Hubs sink to use the built-in hashing for partition keys, the sink must be configured using the `debezium.sink.eventhubs.hashmessagekey` configuration property set to `true` (the default is false).
+When enabled, the `debezium.sink.eventhubs.hashmessagekeyfunction` can be set to `java` (the default), `md5`, `sha1`, or `sha256` to determine the type of hashing algorithm that's used.
+
+==== NATS Jetstream - Asynchronous event dispatch
+
+The Debezium Server NATS Jetstream sink adapter previously relied on synchronous publishing, which imposes performance limitations for high volume streaming scenarios.
+In order to support high volume streaming scenarios, the NATS sink supports two new configuration options (https://issues.redhat.com/browse/DBZ-9474[DBZ-9474]):
+
+|===
+|Property |Default |Description
+
+|`async.enabled`
+|`true`
+|Specifies whether the sink adapter should use asynchronous publishing.
+
+|`async.timeout.ms`
+|`5000`
+|The number of milliseconds before an asynchronous publish call fails due to timeout.
+|===
+
+In this release, the new asynchronous publishing feature is enabled by default but can be disabled by setting `async.enabled` to `false`.
+
+[NOTE]
+====
+All Debezium sink adapter configurations have a unique prefix.
+Be sure to prefix these new configurations with `debezium.sink.nats-jetstream.` if you need to customize their defaults.
+====
+
+
+[id=new-features-and-improvements-platform]
+=== Debezium Platform
```

Comment:
I would add also 

https://github.com/debezium/debezium-platform/pull/90
https://github.com/debezium/debezium-platform/pull/81

Mainly the connection support.